migration_toolkit_for_containers/index.adoc

```diff
@@ -0,0 +1,79 @@
+[id="migration-toolkit-for-containers-overview"]
+= Migration toolkit for containers overview
+include::modules/common-attributes.adoc[]
+:context: migration-toolkit-for-containers-overview
+
+toc::[]
+
+You can migrate stateful application workloads between {product-title} 4 clusters at the granularity of a namespace by using the Migration Toolkit for Containers (MTC). To learn more about MTC see xref:../migration_toolkit_for_containers/about-mtc.adoc#about-mtc[Understanding MTC].
+
+[discrete]
+[id='mtc-overview-install-mtc']
+== MTC installation
+xref:../migration_toolkit_for_containers/installing-mtc.adoc#installing-mtc[MTC installation]: You can install the Migration Toolkit for Containers (MTC) on {product-title} 4. After installing MTC, you must configure an object storage to use as a replication repository. You must install the Migration Toolkit for Containers that is compatible with your {product-title} version by using OLM. You manually install the legacy Migration Toolkit for Containers Operator on {product-title} 4.5 and earlier versions. You can check the following items:
+
+* xref:../migration_toolkit_for_containers/installing-mtc.adoc#installing-mtc[Migration Toolkit for Containers Operator on {product-title}]
+* xref:../migration_toolkit_for_containers/installing-mtc.adoc#migration-configuring-proxies_installing-mtc[Proxies]
+* xref:../migration_toolkit_for_containers/installing-mtc.adoc#configuring-replication-repository_installing-mtc[Replication repository]
+* xref:../migration_toolkit_for_containers/installing-mtc.adoc#migration-configuring-mcg_installing-mtc[Multicloud Object Gateway]
+
+You can also xref:../migration_toolkit_for_containers/installing-mtc-restricted.adoc#installing-mtc-restricted[install the MTC in a restricted network environment].
+
+[NOTE]
+====
+If you are migrating from {product-title} 3, see xref:../migrating_from_ocp_3_to_4/about-migrating-from-3-to-4.adoc#about-migrating-from-3-to-4[About migrating from {product-title} 3 to 4] and xref:../migrating_from_ocp_3_to_4/installing-3-4.adoc#migration-installing-legacy-operator_installing-3-4[Installing the legacy {mtc-full} Operator on {product-title} 3].
+====
+
+[discrete]
+[id='mtc-overview-upgrade-mtc']
+== Upgrade MTC
+xref:../migration_toolkit_for_containers/upgrading-mtc.adoc#upgrading-mtc[Upgrade MTC]: You can upgrade the MTC by using the Operator Lifecycle Manager.
+
+
+[discrete]
+[id='mtc-overview-mtc-checklists']
+== MTC checklists
+xref:../migration_toolkit_for_containers/premigration-checklists-mtc.adoc#premigration-checklists-mtc[MTC checklists]: Before you migrate your application workloads with the Migration Toolkit for Containers (MTC), you can review the following checklists.
```

Comment:
following checklists > premigration checklists.

Then remove the bullet list of checklists. Not necessary.